tests/unit/plugins/janitor/test_gpubsub_publisher.py

```diff
@@ -101,27 +107,34 @@ def gpubsub_publisher_inst(monkeypatch, kwargs):
 
     gpubsub_publisher_inst.publisher.publish.assert_called_once_with(
         gpubsub_publisher_inst.topic, bytes_msg1)
-    assert 1 == len(gpubsub_publisher_inst._messages)
+    assert expected_num_messages == len(gpubsub_publisher_inst._messages)
 
 
-@pytest.mark.parametrize('raises,exp_log_records', [
-    [False, 1],
-    [Exception('foo'), 2],
+@pytest.mark.parametrize('raises', [
+    False,
+    Exception('foo'),
```

Comment:
This should be True, or else the line below shouldn't have the exception hardcoded.